build/stf-run-ci/tasks/setup_base.yml

```diff
@@ -28,51 +28,84 @@
         targetNamespaces:
         - "{{ namespace }}"
 
-- block:
-  - name: Create openshift-cert-manager-operator namespace
-    k8s:
-      definition:
-        apiVersion: project.openshift.io/v1
-        kind: Project
-        metadata:
-          name: openshift-cert-manager-operator
-        spec:
-          finalizers:
-          - kubernetes
+- when: not __deploy_from_index_enabled | bool
+  block:
+    - name: Create openshift-cert-manager-operator namespace
+      k8s:
+        definition:
+          apiVersion: project.openshift.io/v1
+          kind: Project
+          metadata:
+            name: openshift-cert-manager-operator
+          spec:
+            finalizers:
+            - kubernetes
 
-  - name: Create openshift-cert-manager-operator OperatorGroup
-    k8s:
-      definition:
-        apiVersion: operators.coreos.com/v1
-        kind: OperatorGroup
-        metadata:
-          name: openshift-cert-manager-operator
-          namespace: openshift-cert-manager-operator
-        spec: {}
+    - name: Create openshift-cert-manager-operator OperatorGroup
+      k8s:
+        definition:
+          apiVersion: operators.coreos.com/v1
+          kind: OperatorGroup
+          metadata:
+            name: openshift-cert-manager-operator
+            namespace: openshift-cert-manager-operator
+          spec: {}
 
-  - name: Get OCP version
-    shell: oc version -o yaml | grep openshiftVersion | awk '{print $2}'
-    register: ocp_ver
+    - name: Get OCP version
+      shell: oc version -o yaml | grep openshiftVersion | awk '{print $2}'
+      register: ocp_ver
```

Comment:
I might move this out of here and to main.yml because it might be useful to use for ose-registry-image references or other situations where OCP images are version specific.

Reply:
Yes, please. ocp_ver is useful to have everywhere, really.